src/Sylius/Behat/Client/ResponseChecker.php

```diff
@@ -55,7 +55,7 @@ public function getError(Response $response): string
 
     public function isCreationSuccessful(Response $response): bool
     {
-        return $response->getStatusCode() === Response::HTTP_CREATED;
+        return $response->getStatusCode() === Response::HTTP_CREATED || $response->getStatusCode() === Response::HTTP_OK;
```

Comment:
`Response::HTTP_OK` should not be checked in this method, as the method has name `isCreationSuccessful`